src/expr/src/vector_op/trigonometric.rs

```diff
@@ -282,26 +362,4 @@ mod tests {
             two * atan_f64(y / (F64::from(F64::from(x.powi(2) + y.powi(2)).sqrt()) + x)),
         )
     }
-
-    #[test]
-    fn test_degrees_and_radians() {
```

Comment:
degrees and radians already implemented in https://github.com/risingwavelabs/risingwave/pull/9108